### ISSUE TITLE
Message to community: Spark collaborator needed

### ISSUE THREAD

**Issue by perwendel** [perwendel/spark#1155]
Hi all,

First I would like to apologize for the recent inactivity. There are several reasons but the main one is that I've started studying at the university again. I thought I would be able to handle both that on the Spark project but it's not been the case.

Therefore, I want to find someone that wants to join the Spark project in the role of both developer, designer and maintainer. I would continue to help with API design and approval (at least for a period).

If you're interested email me at per.i.wendel@gmail.com and tell med a little bit about yourself.

Cheers,
Per

**Comment by axel1200** [comment]
Maybe setting up some way to communicate gets more people into the project, was thinking of something like https://spectrum.chat/ ... just an idea

**Comment by stephanmg** [comment]
So bottom line is that the project is not maintained currently anymore? I've looked at Spark because I might would consider it for a fun project I'm looking at right now. Otherwise maybe I need to go with React/JS Kotlin maybe.

**Comment by axel1200** [comment]
There are other frameworks for java micro services tho. E.g. https://micronaut.io/

**Comment by stephanmg** [comment]
@axel1200 is this a comparable alternative to spark?

**Comment by axel1200** [comment]
Well I guess so

**Comment by valtteric** [comment]
@stephanmg https://javalin.io/ is for sure an alternative. However I do have a few quite serious projects currently running on spark, would be super good to hear if I should start planning migrating those to something else. Hope not, but that happens in the open source space. Hard way to learn to avoid tight coupling.

**Comment by Typografikon** [comment]
Microservices are trendy these days so there are many project targeting to similar usecase... In addition to https://micronaut.io/ you should also check https://quarkus.io and maybe also https://vertx.io Anyway it will be nice to have some thorough comparison of java microframeworks. 

I found these overviews https://www.e4developer.com/2018/06/02/the-rise-of-java-microframeworks/ and https://github.com/ngEdmundas/awesome-technology/wiki/Java-Microframeworks

**Comment by MikeMitterer** [comment]
Not to forget KTor: https://ktor.io/

**Comment by Doogiemuc** [comment]
This is a really interesting thread. I hope sparkjava will continue and find developers.
I worked with most of the mentioned frameworks. They all have their pro's and con's    sparkjava is by far the most simple one to use (and to learn!)   On the other hand micronaut is the biggest and most complex one.   Its more an alternative to spring.

**Comment by axel1200** [comment]
I agree. We taught Spark Java in the university programming course and it was perfect. It has a small documentation and is super easy to use.

**Comment by Clivern** [comment]
i would like to help here but seems PRs not reviewed anymore.

**Comment by buckelieg** [comment]
> i would like to help here but seems PRs not reviewed anymore.

I think it is quite a bunch of people that want to continue developing this project. I think we should try to switch to [this](https://github.com/Intellisrc/spark) fork since its owner updates it regularly.
